cfme/intelligence/chargeback/assignments.py

```diff
@@ -124,19 +126,19 @@ def step(self):
         )
 
 
-@navigator.register(Assign, 'Storage')
+@navigator.register(AssignsCollection, 'Storage')
 class AssignStorage(CFMENavigateStep):
-    prerequisite = NavigateToSibling('All')
+    prerequisite = NavigateToAttribute('parent', 'All')
```

Comment:
The parent of an AssignsCollection will be the appliance, the navigation prerequisite doesn't make sense to me, please explain intent here?

Reply:
@mshriver This change was made based on the review by @rlbabyuk in the first few comments on this PR. It can be seen here https://www.dropbox.com/s/c0pw0qon1zv3fv5/Screenshot_20180822015452.jpg?dl=0 :)

Reply:
When she made that comment, the navigation class was registered against the entity class, not the collection class.  The wording in her comment is still accurate - 'All' is a destination for the collection class (and thus 'parent' was needed when registered against the entity).

Have you tested these navigation destination directly in ipython? Assuming you'll hit NavigationDestinationNotFound exception.

Reply:
@mshriver Didn't know about that. Sorry if this doubts are trivial . I've recently joined as an intern :)

Reply:
Ah okay that explains why I can't quite find you :) 

No worries, there are a lot of complex components within the framework when it comes to modeling the pages and registering navigation, views, and collections/entities together.  I appreciate your persistence and willingness to accept constructive review comments with this PR, looking very good now.

Reply:
Thanks a ton for the encouragement!